### ISSUE TITLE
Support for custom FPS caps within the toolbar menu

### ISSUE THREAD

**Issue by Crystalflxme** [axstin/rbxfpsunlocker#1938]
I'm aware you can set a custom FPS cap within the settings file, but I wish there was a way to add custom options under the FPS Cap menu.

A possible change could be the addition of an array in the settings file of a list of custom options for FPS caps. [The UI code](https://github.com/axstin/rbxfpsunlocker/blob/master/Source/ui.cpp) could go though the array and create the buttons accordingly.

For example: The settings file could look like this:
```
UnlockClient=1
UnlockStudio=1
FPSCapSelection=0
FPSCap=0.000000
CustomFPSCaps={25, 315, 150}
CheckForUpdates=1
NonBlockingErrors=1
SilentErrors=0
QuickStart=0
```

[The settings code](https://github.com/axstin/rbxfpsunlocker/blob/master/Source/settings.cpp), which goes through the settings and loads them, could also initialize the CustomFPSCaps key into an array. Then, on startup, or when the load settings button is pressed, [the UI code](https://github.com/axstin/rbxfpsunlocker/blob/master/Source/ui.cpp) could load them into the dropdown/selection menu. This would result in the following or something similar:

![image](https://user-images.githubusercontent.com/43885283/103796539-50699780-5082-11eb-93ac-5536a07f5e61.png)
*(Sorry for the bad art)*

Apologies if I misinterpreted what some of the code I mentioned is supposed to do. I'm not extremely familiar with C++ code.

**Comment by axstin** [comment]
I very much appreciate the quality of your issue (compared to the many others submitted to this repository). I'll look into adding this in the next update.

**Comment by Tacki0n4** [comment]
I agree I want you to add custom Fps because I need below 30 fps for a glitch and using the graphics mode isn't reliable
